src/main/scala/com/cognite/sdk/scala/v1/resources/threeD.scala

```diff
@@ -178,8 +177,11 @@ class ThreeDRevisions[F[_]](val requestSession: RequestSession[F], modelId: Long
       Constants.defaultBatchSize
     )
 
-  override def retrieveByIds(ids: Seq[Long]): F[Seq[ThreeDRevision]] =
-    RetrieveByIds.retrieveByIds(requestSession, baseUri, ids)
+  def retrieveById(id: Long): F[ThreeDRevision] = {
```

Comment:
I'd prefer to still extend `RetrieveByIds` and implement `retrieveByIds` in terms of `retrieveById`, something like (untested)
```scala
override def retrieveByIds(ids: Seq[Long]): F[Seq[ThreeDRevision]] =
  ids.map(retrieveById).traverse(identity)
```